packages/components/src/components/MessageStripe/MessageStripe.tsx

```diff
@@ -34,11 +38,13 @@ export const MessageAction = ({
 const backgroundVariants: Record<Variant, string> = {
   trial: '#644ED7',
   warning: '#F7CC66',
+  primary: 'button.background',
```

Comment:
Lovely `'button.background'` 😄. You can leave this for now, but in the future I think we want to convert the values in this codebase to hex to make it slightly easier to turn it into proper design system components. Just a thought that popped up when reviewing.